spec/features/review_card_spec.rb

```diff
@@ -14,8 +14,8 @@
 
   scenario 'enter the incorrect translation' do
     sign_in(user)
-    fill_in 'input_text', with: 'Приветик'
+    fill_in 'input_text', with: 'Пюратрет'
     click_on 'Проверить'
-    expect(page).to have_content 'Неправильный перевод'
+    expect(page).to have_content 'Неправильно.В слове ..Привет. Вы допустили 4 ошибок'
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.